### DISCUSSION TITLE
.md files and internet search engines

### DISCUSSION THREAD

**Discussion by clmarchesan** [a-cosmology-group/acg#23] [General]
From my vague memories, to index an htm page, a search engine like Google started from the Title tag and then checked whether the body of the page was consistent with the title. It is still so? And for GitHub how does it work?

Regards,
Claudio

**Comment by ftessier** [comment]
I don't know. I believe indexing the internet has evolved to algorithms that I would not even begin to comprehend. 🤓

As a major hub of the world's source code, I trust GitHub is indexed thoroughly and frequently. My experience has been consistent with that view.

**Comment by RedshiftDrift** [comment]
> From my vague memories, to index an htm page, a search engine like Google started from the Title tag and then checked whether the body of the page was consistent with the title. It is still so? And for GitHub how does it work?

It does not work like that anymore.  Focus on the science and don't worry, once the `cosmology.info` URL points to the GitHub webpage, Google will re-index the entire GitHub website with everybody's pages.  The `cosmology.info` URL is on Google's search engine list since 2018 (I used *Google search console* to make sure Google indexes ACG).
